server/RafRate.Data/Entities/Users/UserEntity.cs

```diff
@@ -2,18 +2,8 @@
 
 namespace RafRate.Data.Entities;
 
-// will be replaced with Identity
 public class UserEntity : IEntity
```

Comment:
Why you not create custom `IdentityUser`? It will be needed when you will map user with different entities